binding/java/src/ai/picovoice/rhino/Rhino.java

```diff
@@ -127,7 +127,7 @@ public RhinoInference getInference() throws RhinoException {
             reset(libraryHandle);
 
             return inference;
-        } catch (Exception e) {
+        } catch (RhinoException e) {
```

Comment:
You don't even need to catch here. The function is marked `throws RhinoException` so you can remove the try/catch entirely